src/main/java/com/GeorgesServer/app/Router.java

```diff
@@ -3,17 +3,15 @@
 import com.GeorgesServer.app.com.GeorgesServer.handler.*;
 import com.GeorgesServer.app.com.GeorgesServer.request.ClientRequest;
 
-import java.util.HashMap;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
 public class Router {
 
-    private HashMap<String, IHandler> routes;
-
-    public Router() {
-        routes = new HashMap<>();
-    }
-
     public IHandler route(String publicFolderPath, ClientRequest request) {
```

Comment:
I forgot to mention this in the other PRs, but your implementation of Router is specific to CobSpec and thus not reusable. Routing is behavior that's consistent across server applications so I'd expect it to live in some library/package along with the rest of the implementation agnostic code.